File: chap11/exercise/ex2.R
library(cmdstanr)

d <- read.csv('../input/data-season.csv')
T <- nrow(d)
data <- list(T=T, Y=d$Y, L=4, Tp=8)

model <- cmdstan_model('ex2.stan')
fit <- model$sample(data=data, seed=123, parallel_chains=4)
fit$save_object(file='ex2.RDS')

File: chap11/exercise/ex2.py
import pandas
import cmdstanpy

d = pandas.read_csv('../input/data-season.csv')
T = len(d)
data = {'T':T, 'Y':d.Y, 'L':4, 'Tp':8}

model = cmdstanpy.CmdStanModel(stan_file='ex2.stan')
fit = model.sample(data=data, seed=123, parallel_chains=4)
fit.save_csvfiles('result-ex2')
